llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -2037,3 +2037,31 @@ define void @foo(i32 %arg) {
   }
   EXPECT_EQ(NewPHI->getNumIncomingValues(), PHI->getNumIncomingValues());
 }
+
+TEST_F(SandboxIRTest, UnreachableInst) {
+  parseIR(C, R"IR(
+define void @foo() {
+  ret void
+  unreachable
+}
+)IR");
+  llvm::Function *LLVMF = &*M->getFunction("foo");
+  sandboxir::Context Ctx(C);
+  sandboxir::Function *F = Ctx.createFunction(LLVMF);
+  auto *BB = &*F->begin();
+  auto It = BB->begin();
+  auto *Ret = &*It++;
+  auto *UI = cast<sandboxir::UnreachableInst>(&*It++);
+
+  EXPECT_TRUE(llvm::isa<sandboxir::UnreachableInst>(UI));
+  // Check create(InsertBefore)
```

Comment:
Let's also check:
```
EXPECT_EQ(UI->getNumSuccessors(), 1u);
EXPECT_EQ(UI->getNumOfIRInstrs(), 1u);
```

Reply:
The `getNumSuccessors()` member function in `llvm::UnreachableInst` returns a 0. Should I make do 
`EXPECT_EQ(UI->getNumSuccessors(), 0u);` instead?

Reply:
Oops yes, that's what I meant to write :)

Reply:
All done :)